test/fuzz/checksum_fuzzer.c

```diff
@@ -13,72 +13,73 @@
 #  include "zlib-ng.h"
 #endif
 
-int LLVMFuzzerTestOneInput(const uint8_t *data, size_t dataLen) {
-  uint32_t crc0 = PREFIX(crc32)(0L, NULL, 0);
-  uint32_t crc1 = crc0;
-  uint32_t crc2 = crc0;
-  uint32_t adler0 = PREFIX(adler32)(0L, NULL, 0);
-  uint32_t adler1 = adler0;
-  uint32_t adler2 = adler0;
-  uint32_t combine1, combine2;
-  /* Checksum with a buffer of size equal to the first byte in the input. */
-  uint32_t buffSize = data[0];
-  uint32_t offset = 0;
-  uint32_t op[32];
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t dataLen)
+{
```

Comment:
We try to use { at the same line as the function definition, so not sure why you changed it this one place..

Reply:
Done. I have also made all the other functions the same with the brace on the same line as the function definition.